resources/models/resource.py

```diff
@@ -205,6 +205,7 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                                   null=True, blank=True)
     reservation_metadata_set = models.ForeignKey('resources.ReservationMetadataSet', null=True, blank=True,
                                                  on_delete=models.SET_NULL)
+    external_calendar_url = models.URLField(verbose_name=_('External calendar url'), null=True, blank=True)
```

Comment:
Rename this to `external_reservation_url` with a `help_text` of `A link to an external reservation system if this resource is managed elsewhere`.

Reply:
Resolved in eb4340f.